include/range/v3/view/iota.hpp

```diff
@@ -524,16 +543,17 @@ namespace ranges
                 detail::take_exactly_view_<iota_view<Val>, true>>
             ints_fn::operator()(Val from, Val to) const
             {
-                return {iota_view<Val>{from}, detail::iota_minus_(to, from)};
+                return {iota_view<Val>{from}, detail::ints_distance_(from, to)};
             }
 
             struct closed_ints_fn
             {
                 template<typename Val,
                     CONCEPT_REQUIRES_(Integral<Val>())>
+                RANGES_DEPRECATED("view::closed_ints is deprecated; use view::closed_indices instead!")
                 detail::take_exactly_view_<iota_view<Val>, true> operator()(Val from, Val to) const
                 {
-                    return {iota_view<Val>{from}, detail::iota_minus_(to, from) + 1};
+                    return {iota_view<Val>{from}, detail::ints_distance_(from, to) + 1};
```

Comment:
Is this why `ints_distance_` forbids `to - from == max`? If so, that precondition should be here instead of there.